rules/CodingStyle/ClassNameImport/ClassNameImportSkipVoter/ClassLikeNameClassNameImportSkipVoter.php

```diff
@@ -47,7 +47,7 @@ public function shouldSkip(File $file, FullyQualifiedObjectType $fullyQualifiedO
 
         $shortNameLowered = $fullyQualifiedObjectType->getShortNameLowered();
         $fullyQualifiedObjectTypeNamespace = strtolower(
-            substr($fullyQualifiedObjectType->getClassName(), 0, -strlen($fullyQualifiedObjectType->getShortName()) - 1)
+            substr($fullyQualifiedObjectType->getClassName(), 0, -strlen($fullyQualifiedObjectType->getShortName()) - 1) ?: ''
```

Comment:
Can you add a note about PHP 7.4? So we don't accidentally remove it :)

Reply:
ohh I just see rector-src does not run PHPStan on PHP7.x.. you are only using a single PHP 8.2 job and therefore miss PHPStan errors on all other PHP versions